### PR TITLE
Chapter 11: Account Activation

### PR DESCRIPTION
**Pull request by trungnguyen269**
image

### CODE REVIEW

**Comment by trungnguyen269** [comment]
ready

File: app/assets/javascripts/account_activations.coffee [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# Place all the behaviors and hooks related to the matching controller here.
+# All this logic will automatically be available in application.js.
+# You can use CoffeeScript in this file: http://coffeescript.org/
```
Comment (chien-pv) [inline app/assets/javascripts/account_activations.coffee]: Xóa file

File: app/assets/stylesheets/account_activations.scss [hunk 2]
```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the AccountActivations controller here.
+// They will automatically be included in application.css.
+// You can use Sass (SCSS) here: http://sass-lang.com/
```
Comment (chien-pv) [inline app/assets/stylesheets/account_activations.scss]: Xóa file

File: app/helpers/account_activations_helper.rb [hunk 3]
```diff
@@ -0,0 +1,2 @@
+module AccountActivationsHelper
+end
```
Comment (chien-pv) [inline app/helpers/account_activations_helper.rb]: Xóa file

**Comment by chien-pv** [comment]
commented

**Comment by trungnguyen269** [comment]
ready

**Comment by trungnguyen269** [comment]
ready

**Comment by chien-pv** [comment]
merged

**Pull request merged**